packages/azure-services/package.json

```diff
@@ -41,7 +41,8 @@
         "typemoq": "^2.1.0",
         "typescript": "^3.8.3",
         "webpack": "^4.42.0",
-        "webpack-cli": "^3.2.3"
+        "webpack-cli": "^3.2.3",
```

Comment:
unit tests are taking more time with this pr - (around 10 min)
Can you check by running the jest-slow-run-reporter as mentioned in https://github.com/microsoft/accessibility-insights-service/issues/601 (just uncomment the commented code in https://github.com/microsoft/accessibility-insights-service/blob/8102de4bee93e9355cfd5097bb087dcb64fb7b13/jest.config.base.js#L56 & run.